docs/core/compatibility/containers/8.0/aspnet-port.md

```diff
@@ -7,27 +7,64 @@ ms.date: 07/12/2023
 
 The default ASP.NET Core port configured in .NET container images has been updated from port 80 to 8080.
 
-We also added the new `ASPNETCORE_HTTP_PORTS` environment variable instead of `ASPNETCORE_URLS`.
+We also added the new `ASPNETCORE_HTTP_PORTS` environment variable as a simpler alternative to `ASPNETCORE_URLS`. The new variable expects a semicolon delimited list of ports numbers, while the older variable expects a more complicated syntax.
 
 ## Previous behavior
 
 Prior to .NET 8, you could run a container expecting port 80 to be the default port and be able to access the running app.
 
-For example, running the following command allowed you to access the app locally at port 9999, which is mapped to port 80 in the container:
+For example, running the following command allowed you to access the app locally at port 8000, which is mapped to port 80 in the container:
```

Comment:
nit, would something without 8's read better? eg, 1234

8000 looks a lot like 8080 if it's small...

Reply:
and 8000 again below

Reply:
It had `9999` before but that's not idiomatic. I used something that is idiomatic. It's a tradeoff.

Reply:
is there a 4 digit idiomatic HTTPx port that's not 8xxx ? 

but no matter...

Reply:
ha, seems like the only other one is 8080.. that's worse

https://www.iana.org/assignments/service-names-port-numbers/service-names-port-numbers.xhtml?search=http-alt

Reply:
Yup. This is why I use `8000`. There is a minimum level of "attention to detail" that applies to using containers and this aspect is within that.